powershell/llcsharp/schema/string.ts

```diff
@@ -200,11 +200,11 @@ ${this.validateEnum(eventListener, property)}
     return `await ${eventListener}.AssertMaximumLength(${nameof(property.value)},${property},${len});`;
   }
   private validateRegex(eventListener: Variable, property: Variable): string {
-    const pattern = (<any>this.schema).pattern;
+    const pattern = (<any>this.schema).pattern?.replace('"', '""');
```

Comment:
Is it possible that there are more than one double quote in pattern? Changing to replace(/"/g, '""') may be better.